source/_docs/terminus/plugins/create.md

```diff
@@ -184,122 +184,141 @@ The first digit of a plugin version should always the Terminus MAJOR version the
 If your plugin has a minimum required version of Terminus, you can specify that in the `compatible-version` section of `composer.json`. You can use the [standard composer version constraints syntax](https://getcomposer.org/doc/articles/versions.md). If you do change `compatible-version`, please make sure that your constraint expression does not accidentally include the next major version of Terminus. In other words, `>=1.3 <2.0.0` is fine but `>=1.3` is not.
 
 ## Testing Plugins
-Adding automated testing is an important step to take for plugins that are being distributed. Automated tests give prospective new users the assurance that the plugin is still working, and also provide a basis for evaluating changes to the plugin.
+Adding automated testing is an important step to take for distributed plugins. Automated tests give prospective new users the assurance that the plugin works, and provides a basis for evaluating changes to the plugin.
```

Comment:
You improved the English by changing this to `distributed plugins`, but now it might ambiguously imply that the plugins run on multiple machines due to overloading of the word `distributed`. Not sure if this would actually confuse anyone or not.